python/py-nclib/Portfile

```diff
@@ -21,12 +21,12 @@ categories-append    net
 platforms            darwin
 maintainers          {@F30 f30.me:f30} openmaintainer
 
-python.versions      27 34 35 36
+python.versions      27 34 35 36 37
 
 master_sites         pypi:n/nclib
 distname             nclib-${version}
-checksums            rmd160  978d85bc2520a6836674b5deaaca5c93caf45eea \
-                     sha256  a4435b0e5fe5302326b1c1a6be430c45ae9d5c3e9de2103381780121651e101e
+checksums            rmd160  a1044012c12cba1ced18bc493f732ab6fc18b056 \
+                     sha256  7ee56ca74ade02796c01923b117d7192b7a2381ab06464b4168b0f4e0e0deb1d
```

Comment:
We now add the `size` parameter to `checksums`.

Reply:
I have now added the `size` parameter.